about.html

```diff
@@ -18,7 +18,7 @@
                 <a id="closebutton" href="#"><i class="fa-solid fa-x fa-xl closer"></i></i></a>
             </div>
             <div class="menuitems">
```

Comment:
Optional -
I would be preferable if you video link is in your projects readme file, not the Pull request, please try making those tiny changes and your project will be perfect 😄 👍🏾  🥇